changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 10.1.0 - xxxx-xx-xx =
+* Update - Remove `redirect_url` parameter from ECE payment flow
```

Comment:
Two nits:
* I think this qualifies as a Fix rather than an update - there is no developer or user-facing change AFAICT.
* I think it would be better to use `Express Checkout` instead of ECE for general readability
```suggestion
* Fix - Remove `redirect_url` parameter from Express Checkout payment flow
```